app/include/rom.h

```diff
@@ -126,6 +126,8 @@ int ets_strncmp (const char *s1, const char *s2, size_t n);
 void ets_delay_us (uint32_t us);
 
 int ets_printf(const char *format, ...)  __attribute__ ((format (printf, 1, 2)));
+int os_printf(const char *format, ...) __attribute__ ((format (printf, 1, 2)));
+int os_printf_plus(const char *format, ...)  __attribute__ ((format (printf, 1, 2)));
```

Comment:
The os_printf\* ones should go into the `osapi.h` override header in `sdk-overrides/include/`; They're not actually provided in the ESP8266 ROM.